### PR TITLE
prepare for release

### PR DESCRIPTION
**Pull request by cfsnook**

### CODE REVIEW

File: ac.soton.eventb.statemachines.animation.feature/feature.xml [hunk 1]
```diff
@@ -17,23 +17,23 @@
    </copyright>
 
    <url>
-      <update label="%updateSiteName" url="http://rodin-b-sharp.sourceforge.net/updates/site.xml"/>
+      <update label="%updateSiteName" url="https://eventB-soton.github.io/updateSite/site.xml"/>
    </url>
```
Comment (tshoang) [inline ac.soton.eventb.statemachines.animation.feature/feature.xml:22]: There is no site.xml for "eventB-soton.github.io/updateSite"

**Review by tshoang — CHANGES_REQUESTED** [review]
Can you change the Soton Update Site URL to remove "site.xml"?

File: ac.soton.eventb.statemachines.animation.feature/feature.xml (hunk 1 above)
Comment (cfsnook) [reply ac.soton.eventb.statemachines.animation.feature/feature.xml:22]: thanks.. well spotted

**Review by tshoang — APPROVED** [review]
Great

**Pull request merged**